mobile_back_samsung/cpp/eden_mlperf_c_backend/jni/eden_sdk/include/samsung_settings.h

```diff
@@ -96,6 +96,7 @@ benchmark_setting {
   accelerator_desc: "npu"
   configuration: "Samsung Exynos"
   src: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/Samsung/is.nnc"
+  single_stream_expected_latency_ns: 900000
```

Comment:
The vendor can change the default value of 1M by explicitly set it in the benchmark_setting

Reply:
Group policy discussion is we need to enforce that this parameter does not exceed 1M, to avoid manipulation for slower tasks, please modify the PR to take min(parameter, 1M) @anhappdev

Reply:
Should the app throw an exception if the latency > 1M (which I prefer) or just silently take min(parameter, 1M)?

Reply:
I think just silently take min(parameter, 1M) to avoid disruption, and in settings file we can put a comment to reflect that

Reply:
If the app just silently takes min(parameter, 1M), other / new engineers who not know about this logic will wonder why latency > 1M has no effect. Each backend has a separated setting file, so there will be many comments to add and (if they are noted is another question).
Throw an exception would be a more correct way to handle this, IMO. If someone wants to change the default latency, they will run a test to see how it works and knows exactly what to fix if the latency > 1M.

Reply:
Yes, throwing an exception sounds more correct.

Reply:
Done